src/richie/apps/search/defaults.py

```diff
@@ -0,0 +1,54 @@
+from django.conf import settings
+from django.forms import ChoiceField
+
+import arrow
+
+
+# Define our aggregations names, for our ES query, which will match with the field
+# names on the objects & the facets we return on the API response
+RESOURCE_FACETS = getattr(
+    settings, "RICHIE_SEARCH_RESOURCE_FACETS", ["organizations", "subjects"]
+)
+
+FILTERS_HARDCODED_DEFAULT = {
+    "availability": {
+        "field": ChoiceField(
+            choices=[("coming_soon", ""), ("current", ""), ("open", "")],
```

Comment:
Why empty human names ?

Reply:
There is literally no use for these human names, they are not displayed anywhere not useful for anything as far as I can tell.

I think they're just required because the Django field classes were built with actual generated HTML forms in mind, which would need human names.